manifests/params.pp

```diff
@@ -12,6 +12,10 @@
   $open_ssl_listen_port      = undef
   $postgres_listen_addresses = 'localhost'
 
+  $certificate_whitelist_file = '/etc/puppetdb/certificate-whitelist'
```

Comment:
This path is no longer what we use for the latest version of the module - that is, be careful to lift the path from other variables, not static like this - this isn't a good location to store it for PuppetDB 3.x for example.